README.md

```diff
@@ -28,18 +28,28 @@ This repository contains a Bash script that parses an Nginx log file (`nginx.log
 
 2. **Basic Parsing (Method 2 - Using `grep`):**
     ```bash
-   ./script.sh -f nginx.log -m 2
+   ./script.sh -m 2
    ```
 
-3. **Adding Country Information (Hard Level):**
-   The script fetches the country code for each IP address using the `ipinfo.io` API:
+3. **Basic Parsing (Method 3 - Using `IPinfo grepip`):**
    ```bash
-   ./script.sh -f nginx.log -m 1 -c
+   ./script.sh -m 3
    ```
```

Comment:
I like that you added a new method. However, you did not add any information that I need to install this tool before using it. Please fix this